micrometer-core/src/main/java/io/micrometer/core/instrument/push/PushMeterRegistry.java

```diff
@@ -136,7 +136,12 @@ public void close() {
         super.close();
     }
 
-    private void waitForInProgressScheduledPublish() {
+    /**
+     * Wait until scheduled publishing by {@link PushMeterRegistry} completes, if in
```

Comment:
Should we add a note that this should not be called in most of the cases by the users?

Reply:
Only implementations of `PushMeterRegistry` should have access to call it, so I didn't think about mentioning it. Implementations may need to call it like StepMeterRegistry (or OtlpMeterRegistry). What do you think?

Reply:
Sounds reasonable. 👍🏼